### PR TITLE
topology2: cavs: simplify/clarify topologies

### PR DESCRIPTION
**Pull request by plbossart**

### CODE REVIEW

**Comment by plbossart** [comment]
WIP, no merge for now. More changes coming.

File: tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf [hunk 1]
```diff
@@ -16,7 +16,6 @@ Object.Pipeline {
 	mixout-gain-dai-copier-playback.0 {
 		index 2
 		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.2.1'
```
Comment (ranj063) [inline tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf:14]: yes, if we're not setting the stream name, we can remove the Object completely

**Comment by keqiaozhang** [comment]
SOFCI TEST

File: tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf (hunk 1 above)
Comment (plbossart) [reply tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf:14]: done

**Comment by plbossart** [comment]
SOFCI TEST

**Pull request merged**